video_processing/script.py

```diff
@@ -41,23 +60,24 @@ def get_heart_rate(file_url):
     frame_count = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
     video_duration = frame_count / fps
 
-    print("video_duration: ")
-    print(video_duration)
+    print('video_duration: {}'.format(video_duration))
```

Comment:
jenom takovy python trik jsou ty f_string nebo jak se tomu rika
`f'video_duration: {video_duration}'`
dela to to same, mne osobne to prijde prijemnejsi na psani i na cteni

Reply:
samozrejme ze znam fstringy :DDD
ja mam radsi .format().. hlavne at jsme konzistentni, jinak je mi to jedno, prepsala jsem to na fstringy ;)

Reply:
no tak to jsi sikovna... proc mas radsi .format()? fstring mi prijde uplne o parnik lepsi, ne?